portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactsService.java

```diff
@@ -66,7 +79,7 @@ public GoogleContactsService(Credential credential, JobDataCache jobDataCache) {
   }
 
   @VisibleForTesting
-  static VCard convertPersonToModel(Person person) throws IOException {
+  static VCard convertPersonToModel(Person person) {
```

Comment:
can this be renamed convertPersonToVCard to be consistent w/ the other converter names?